src/sentry/api/invite_helper.py

```diff
@@ -17,50 +16,37 @@
 from sentry.utils import metrics
 from sentry.utils.audit import create_audit_entry
 
-INVITE_COOKIE = "pending-invite"
-COOKIE_MAX_AGE = 60 * 60 * 24 * 7  # 7 days
```

Comment:
@mdtro I was curious if we need to expire the invite details within 7 days of it being set on the request session. This would preserve a similar behaviour from when the invite details was stored as a cookie.